src/frontend/sharedComponents/FormattedData.tsx

```diff
@@ -154,8 +154,9 @@ function getValueLabel(
     // Look up label from field options. This is not necessary for presets
     // created with mapeo-settings-builder@^3.1.0, which will have these options
     // in the translation file, but is needed for older versions of presets
-    const options = convertSelectOptionsToLabeled(field.options);
-    const matchingOption = options.find(option => option.value === value);
+    const matchingOption = field.options?.find(
```

Comment:
is the comment above no longer relevant then? my understanding is that any presets we use for CoMapeo are generated using more recent versions of settings builder. also, just to verify: we're sure that we won't be dealing with trying to import older configs and needing to handle any of that client-side (as the helper originally seemed to needed for?)